### PR TITLE
PrimeNumbers_Sangavi

### PR DESCRIPTION
**Pull request by sangavim**

### CODE REVIEW

File: PrimeNumbers.java [hunk 1]
```diff
@@ -15,12 +15,41 @@ public static void myassert(boolean  x) {
...
 
-
     private static List<Integer> fourDigitPrimes(){
-        //WRITE YOUR CODE HERE
+        int i,num,check;
+        ArrayList<Integer> primes = new ArrayList<>();
+        for (check = 1000; check <= 9999; check++){
+            int count = 0;
+            num = check/2;
+            for(i = 2; i <= num; i++){
```
Comment (chandramouly) [inline PrimeNumbers.java:43]: Can we reuse this code, the core prime # logic is there in both the functions. you can reuse the code by writing a separate function. Also you can break the loop even if one number is divisible, no need to check till n/2 and then add to primes list.

Comment (balamuralim) [reply PrimeNumbers.java:43]: +1

File: PrimeNumbers.java [hunk 2]
```diff
@@ -15,12 +15,41 @@ public static void myassert(boolean  x) {
...
+                if(check % i == 0){
+                    count++;
+                }
+            }
+            if(count == 0){
+                max ++;
+                if(max == 2){
+                    break;
```
Comment (balamuralim) [inline PrimeNumbers.java:30]: You can return the prime number here, instead of break

File: PrimeNumbers.java [hunk 3]
```diff
@@ -15,12 +15,41 @@ public static void myassert(boolean  x) {
     }
 
     private static int secondMaxSixDigitPrime(){
-        //WRITE YOUR CODE HERE
+        int num,i,check,max=0;
+        for (check = 999999; check >= 100000; check--){
```
Comment (balamuralim) [inline PrimeNumbers.java:19]: **Better Readability** - declare MIN =100000 and MAX=999999 and use them in the for loop

**Pull request closed without merging**